geode-core/src/test/java/org/apache/geode/internal/cache/BucketRegionTest.java

```diff
@@ -61,8 +67,12 @@
 import org.apache.geode.internal.cache.tier.sockets.ConnectionListener;
 import org.apache.geode.internal.statistics.StatisticsClock;
 import org.apache.geode.test.fake.Fakes;
+import org.apache.geode.test.junit.rules.ExecutorServiceRule;
 
 public class BucketRegionTest {
+  @Rule
+  public ExecutorServiceRule executor = new ExecutorServiceRule();
```

Comment:
Is there a way to eliminate the addition of this rule so we can use JUnit 5.

Reply:
It's possible that this rule can work with JUnit 5. Try these two steps:
- Add `testImplementation('org.junit.jupiter:junit-jupiter-migrationsupport')` to geode-core's build.gradle file.
- Add `@EnableRuleMigrationSupport` to the test class.